x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/columns.tsx

```diff
@@ -0,0 +1,233 @@
+        const domain = domains[0];
+        const id = escapeDataProviderId(
+          `networkTopNFlow-table-${ip}-${topNFlowType}-${topNFlowDirection}-domain-${domain}`
+        );
+        return (
+          <DraggableWrapper
+            key={id}
+            dataProvider={{
+              and: [],
+              enabled: true,
+              id,
+              name: domain,
+              excluded: false,
+              kqlQuery: '',
+              queryMatch: {
+                field: domainAttr,
+                value: domain,
+              },
+              queryDate: {
+                from: startDate,
+                to: Date.now(),
+              },
+            }}
+            render={(dataProvider, _, snapshot) =>
+              snapshot.isDragging ? (
+                <DragEffects>
+                  <Provider dataProvider={dataProvider} />
+                </DragEffects>
+              ) : (
+                <>
+                  {domain}
+                  {domains.length > 1 && (
+                    <EuiToolTip
+                      content={
+                        <>
+                          {domains.slice(1, 6).map(domainName => (
+                            <span key={`${id}-${domainName}`}>
+                              {defaultToEmptyTag(domainName)}
+                              <br />
+                            </span>
+                          ))}
+                          {domains.slice(1).length > 5 && (
+                            <b>
+                              <br />
+                              more ...
+                            </b>
+                          )}
+                        </>
+                      }
+                    >
+                      <MoreDomains type="eye" />
+                    </EuiToolTip>
+                  )}
+                </>
+              )
+            }
+          />
+        );
+      } else {
+        return getEmptyTagValue();
+      }
+    },
+  },
+  {
+    name: i18n.DIRECTION,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) =>
+      has('network.direction', node)
+        ? get('network.direction', node).length > 0
+          ? get('network.direction', node).map((direction: NetworkDirectionEcs) => (
+              <AddToKql
+                key={escapeDataProviderId(
+                  `networkTopNFlow-table-${topNFlowType}-${topNFlowDirection}-direction-${direction}`
+                )}
+                content={i18n.FILTER_TO_KQL}
+                expression={`network.direction: ${escapeQueryValue(direction)}`}
+                type={type}
+              >
+                {defaultToEmptyTag(direction)}
+              </AddToKql>
+            ))
+          : getEmptyTagValue()
+        : getEmptyTagValue(),
+  },
+  {
+    name: i18n.BYTES,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) =>
+      has('network.bytes', node)
+        ? numeral(node.network!.bytes).format('0.000b')
```

Comment:
We can solve this later and I know we have a ticket for getting the default date time's from the user prefs but just pointing out it looks like the advanced settings also has byte format:

<img width="699" alt="screen shot 2019-02-22 at 4 16 48 pm" src="https://user-images.githubusercontent.com/1151048/53276680-6142d700-36bd-11e9-852b-367a5eed8109.png">

We probably should at least add a backlog item to sweep through before release and fix this area with user prefs unless it's simple to do now.